app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -12,10 +12,11 @@ def initialize
       end
 
       def create_authentication_parameters(authenticator_input)
```

Comment:
do we need the `create_` prefix? can this method be named `authentication_parameters` to match ruby style?

Reply:
Removed create